python-lib/wordcloud_generator.py

```diff
@@ -0,0 +1,193 @@
+# -*- coding: utf-8 -*-
+"""Module with a class to generate wordclouds based on cleaned text"""
+
+import dataiku
+from dataiku.core import dkuio
+import logging
+import matplotlib
+import matplotlib.pyplot as plt
+
+matplotlib.use("agg")
+
+import traceback
```

Comment:
Unused imports

Reply:
Still unused imports ... and one should not mix imports with commands: do all your imports, and then do `matplolib.use`